requirements/prod.txt

```diff
@@ -20,6 +20,7 @@ dennis==0.4.2
 Django==1.6.11
 dj-database-url==0.2.2
 django-cache-nuggets==0.1.1
+django-celery==3.0.23
```

Comment:
Same issue as the `djcelery` app below. Also, I'm fairly certain that djcelery 3.0 is not compatible with Celery 3.1.